docs/project-structure/genezio-configuration-file.md

```diff
@@ -114,6 +125,31 @@ Variables can be used in the scripts. Check the [Usage](#how-to-use-variables-in
 - `local`: `string` | `string[]` **Optional**
 
   A general purpose script that runs before starting the local testing environment.
+### Basic `backend` deployment
+```yaml
+# Learn more about Genezio YAML at https://genezio.com/docs/project-structure/genezio-configuration-file/
+# The name of the project.
+name: (•◡•)project-name(•◡•)
```

Comment:
```suggestion
name: project-name
```

Reply:
Let's remove the emojis - this is a funny unique placeholder that we only use in some internal templates. Shouldn't be in the docs.